src/views/RequestTypeForm/index.tsx

```diff
@@ -31,31 +33,42 @@ const RequestTypeForm = () => {
     }
   );
 
-  const handleCreateIntake = (values: { requestType: string }) => {
+  const handleIntakeCreationRedirect = (intake: SystemIntakeForm) => {
+    switch (intake.requestType) {
+      case 'NEW':
+        history.push(`/governance-task-list/${intake.id}`);
+        break;
+      case 'MAJOR_CHANGES':
+        history.push(`/governance-task-list/${intake.id}`);
+        break;
+      case 'RECOMPETE':
+        history.push(`/governance-task-list/${intake.id}`);
+        break;
+      case 'SHUTDOWN':
+        history.push(`/system/${intake.id}/contact-details`);
+        break;
+      default:
+        console.warn(`Unknown request type: ${intake.requestType}`);
+        break;
+    }
+  };
+
+  const handleCreateIntake = (formikValues: { requestType: string }) => {
+    const formData = {
```

Comment:
This is just the data to create the intake. The only "filled out" value this will ever send is the `requestType`.